src/library.js

```diff
@@ -52,7 +52,6 @@ mergeInto(LibraryManager.library, {
   // JavaScript <-> C string interop
   // ==========================================================================
 
-  $stringToNewUTF8__deps: ['malloc'],
```

Comment:
I think we can keep these, however - @sbc100 's point iirc was that they provide good link-time errors. And some day we may improve linking to use them automatically if we remove deps_info.

Reply:
Yes, please keep these dependencies, for both the reasons mentioned here.